microcosm_caching/memcached.py

```diff
@@ -25,36 +25,30 @@ class SerializationFlag(IntEnum):
     JSON = 2
 
 
-def json_serializer(key, value):
+class JsonSerde:
```

Comment:
Nit: I know this is the parameter name, but it still seems a bit cryptic. Maybe `JsonSerializer`, or `JsonSerializerDeserializer`, or just expand the docstring to use the term "deserializer"?

Reply:
I was following `pymemcache` convention. But definitely the name is a little cryptic.